en/orm/schema-system.rst

```diff
@@ -223,7 +223,7 @@ You can use it to get the list of tables or reflect tables into
     $db = ConnectionManager::get('default');
 
     // Create a schema collection.
-    $collection = $db->schemaCollection();
+    $collection = $db->getSchemaCollection();
```

Comment:
We should also include a comment above this like 'prior to 3.6 use schemaCollection()'